README.md

```diff
@@ -0,0 +1,149 @@
+# omigami
+#### Minimally biased features selection for untargeted metabolomics studies
+
+[![PyPI version shields.io](https://img.shields.io/pypi/v/omigami.svg)](https://pypi.python.org/pypi/omigami) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+<!-- image:: https://img.shields.io/travis/datarevenue-berlin/omigami.svg :target: https://travis-ci.org/datarevenue-berlin/omigami -->
+
+<!-- image:: https://readthedocs.org/projects/omigami/badge/?version=latest :target: https://omigami.readthedocs.io/en/latest/?badge=latest :alt: Documentation Status -->
+
+Easy and powerful tool to select features in untargeted metabolomics studies – with less risk for bias. How?
+Omigami uses multivariate models to perform a recursive feature elimination within a repeated double cross-validation.
+
+## Install
+
```

Comment:
I'm not sure we should remove the mention to virtual environments. If you are not used to python, you probably would not know that before calling PIP you should create one.

Reply:
Well but it's not the responsibility of a python package to explain how to deal with python packages. The user either knows this or he doesn't.